src/components/action-sheet/action-sheet.ts

```diff
@@ -365,8 +369,7 @@ class ActionSheetCmp {
   }
 
   isEnabled() {
-    let tm = this._config.getNumber('overlayCreatedDiff', 750);
-    return (this.created + tm < Date.now());
+    return this.enabled;
```

Comment:
Good idea, but can `isEnabled` be the boolean instead and not even have the `enabled` property?